### DISCUSSION TITLE
Disconnessioni tra HA e Google Home

### DISCUSSION THREAD

**Discussion by federico89mi** [indomus/forum#2823] [HUB: Home Assistant]
Buongiorno,
premetto che utilizzo ormai da quasi 2 anni HomeAssistant sul quale ho integrato i miei 3 google home per i comandi vocali seguendo la guida di InDomus molto utile. Il tutto ha sempre funzionato correttamente, comando circa 40 dispositivi tra luci, tapparelle, condizionatori e caloriferi. 
Il problema che ho sempre avuto e che non ho mai avuto molto tempo per identificarlo è che mi capita molto spesso che HomeAssistant non sia raggiungibile dai Google Home. Per esempio se do un comando vocale ai Google Home chiedendo di spegnermi una luce, 1 volta su 3 circa i google home mi rispondono che "<la mia istanza HA> non è disponibile al momento". 
Non ho molte idee su come identificare il problema. 
L'altro sintomo che ho è che spesso quando tento di accedere ad HA da cell (fuori casa) o anche dentro casa collegato al wifi Ha mi risulta irraggiungibile. Poi dopo qualche secondo torna a funzionare e atterro sulla home. Ho come la sensazione come se Ha andasse in standby se non lo utilizzo da un po' e poi al secondo tentativo di connessione, sia con comandi vocali tramite google home oppure tramite app HA invece poi si ricollega.
Utilizzo una connessione in fibra ftth con router Nexxt Fastweb e HA è installato su un raspberry Pi4 con 8 gb di ram. Non ho altri problemi di connessioni o lentezze del OS HA quindi proprio non ho ideee di come risolvere.
Tramite un post di snakuzzo su indomus da qualche parte avevo provato a disattivare l'ipv6 nelle impostazioni network di Ha. Sembrava aver funzionato per qualche giorno/ora ma poi invece i problemi si sono ripresentati.
Non è nulla di grave in effetti perchè utilizzo regolarmente HA però appunto queste disconnessioni spesso sono fastidiose e non è comodo ogni volta dare i comandi vocali più volte prima che li recepisca.
Qualcuno mi può gentilmente dare una mano?

Grazie a tutti

**Comment by federico89mi** [comment]
nessuno sa aiutarmi?

**Reply by Scialla** [reply]
devi indagare tu.. se te lo fa collegandoti con il nome dominio, e non te lo fa collegandoti localmente con IP, potrebbe essere un problema legato al dominio con cui ti colleghi (duckdns?), altrimenti dovresti indagare spulciandoti i log.

Da qui ci va la sfera di cristallo...

**Reply by federico89mi** [reply]
ok ti ringrazio, ma quali log guarderesti tu per questo tipo di problema?

**Reply by Scialla** [reply]
quindi hai problemi a loggarti anche localmente con IP....
non hai specificato poi che tipo di installazione hai e come raggiungi HA da remoto (tunnel, reverse proxy, etc) e nemmeno se HASSOS, core etc..
cmq mi focalizzerei sull'orario in cui hai il problema e cercherei nel log di sistema di HA intorno a quell'ora, eventualmente mettendo a `debug` il `logger` per quanto riguarda il componente `homeassistant.component.http`
